components/scream/cime_config/buildnml

```diff
@@ -106,21 +106,32 @@ def refine_type(entry):
         return entry
 
 ###############################################################################
-def process_leaves(yaml_entry, case):
+def process_leaf(leaf_str, case):
```

Comment:
Is there a place where you explain briefly what "node" and "leaf" we are talking about? If not, can you add a two liner?

Reply:
Also, a quick explanation of what the 'processing' is, maybe? I think you are processing a single line, replacing cime vars and the if-then-else switches, but might worth explain it (for the sake of maintaining the code).

Reply:
Done. Commit coming soon.